open-api-spec.yaml

```diff
@@ -36,7 +33,7 @@ tags:
   - name: quote
     description: quote operations
 paths:
-  /:
+  '/{id}':
```

Comment:
Why not also call this `accountId` like it is in the other endpoints?

Reply:
I was thinking of consistently making `id` be the resource of the particular endpoint, but I don't feel strongly about it.

Reply:
Tbh, me neither. We can have `id` be the resource of the endpoint.